src/main/java/faang/school/functionalinterfacesandlambdas/metauniverse/NotificationManager.java

```diff
@@ -0,0 +1,30 @@
+package faang.school.functionalinterfacesandlambdas.metauniverse;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class NotificationManager {
+
+    private final Map<String, Consumer<Notification>> notificationHandler = new HashMap<>();
+
+    public void registerHandler(String notification, Consumer<Notification> consumer) {
+        if (notification == null || notification.isBlank()) {
+            System.out.println("Введите тип оповещения");
+        } else {
+            if (consumer == null) {
+                System.out.println("Введите форму оповещения!");
+            } else {
+                notificationHandler.put(notification, consumer);
+            }
+        }
```

Comment:
public void registerHandler(String notification, Consumer<Notification> consumer) {
        if (notification == null || notification.isBlank()) {
            throw new IllegalArgumentException("Notification was not provided");
        }
        if (consumer == null) {
            throw new IllegalArgumentException("Consumer was not provided");
    }

        notificationHandler.put(notification, consumer);
    }